src/components/links/ClubSocialsLinks.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div>
+    <p class="links club-socials">
+      <a :href="links.instagram" target="_blank">
+        <img alt="instagram" src="@/assets/icons/instagram.svg" />
+      </a>
+      <a :href="links.facebook" target="_blank">
+        <img alt="facebook" src="@/assets/icons/facebook.svg" />
+      </a>
+      <a :href="links.medium" target="_blank">
+        <img alt="medium" src="@/assets/icons/medium.svg" />
+      </a>
+    </p>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { ClubSocials } from '@/data/types';
+
+/**
+ * Footer is the website footer.
+ */
+export default Vue.extend({
+  name: 'Footer',
```

Comment:
Should the component name be Footer? or ClubSocialsLink?

Reply:
4153f88